### DISCUSSION TITLE
Does failed trails influence searching space?

### DISCUSSION THREAD

**Discussion by pfk-beta** [optuna/optuna#4931] [Q&A]
Hello,

I'm lookinng for hyper parameters, when using pytorch lib. Sometime, when I use training, I'm getting CUDA error, and trial returns None. I'm wonder if I should remove failed trails from study, to get proper process of searching hyper parameters.

**Comment by nzw0301** [comment]
https://github.com/optuna/optuna/issues/1647 might be a related issue.
